### ISSUE TITLE
Kodi 20.1 no more tv guide

### ISSUE THREAD

**Issue by enigma131** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1123]
⚠️ *Please try the last beta version and check existing issues before issuing a bug.* ⚠️

### Describe the bug
A clear and concise description of what the bug is:
Error message for Guide TV, tested for French channels

### To Reproduce
Steps to reproduce the behaviour:
Select plug-in, navigate through Live TV

### Expected behaviour
A clear and concise description of what you expected to happen:
Obtain Channel's Tv guide

### Screenshots
If applicable, add screenshots to help explain your problem:

### Your environment
 - [ ] Android
 - [ ] iOS
 - [x ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [x ] Windows
 - [ ] Other

 - Operating system version/name:
 - Kodi version: 20.1
 - Catch-up TV & More version: latest beta 21
 - Installation method: normal way

### Debug log
If you can, please add your debug log information (Tutorial available here: https://kodi.wiki/view/Log_file/Easy):

extract:
2023-03-14 09:22:29.265 T:27574   error <general>: [Catch-up-TV-&-More] xmltv module failed with error: 404 Client Error: Not Found for url: https://github.com/Catch-up-TV-and-More/xmltv/raw/master/tv_guide_fr_20230314.xml

**Comment by joaopa00** [comment]
OUi. Je suis en train de tester si je peux remettre l'EPG anglais. Je me donne encore quelques jours pour réussir.  En attendant, les autres EPG ne seront malheureusement pas disponible.

**Comment by khalistoo** [comment]
T'as deja utilise ceci ? https://github.com/iptv-org/api je sais pas si ca vaut le coup de tester l'api ou si ce n'est qu une perte de temps mais en terme de source ca a l aire assez complet

**Comment by joaopa00** [comment]
Je ne connaissais pas. Merci pour l tuyau.

**Comment by joaopa00** [comment]
Devraît être fixé depuis quelques jours. Je ferme. N'hésite pas à réouvrir si ce n'est pas le cas.

**Comment by enigma131** [comment]
Oui merci cela refonctionnne, je n'avais pas remarqué car j'utilise beaucoup les favorits.
